lava-v2-jobs-from-api.py

```diff
@@ -197,20 +195,19 @@ def jinja_render(job):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("--token", help="KernelCI API Token")
+    parser.add_argument("--token", help="KernelCI API Token", required=True)
     parser.add_argument("--api", help="KernelCI API URL", default="https://api.kernelci.org")
     parser.add_argument("--storage", help="KernelCI storage URL", default="https://storage.kernelci.org")
-    parser.add_argument("--jobs", help="absolute path to top jobs folder")
-    parser.add_argument("--tree", help="KernelCI build kernel tree")
-    parser.add_argument("--branch", help="KernelCI build kernel branch")
-    parser.add_argument("--describe", help="KernelCI build kernel git describe")
-    parser.add_argument("--config", help="configuration for the LAVA server")
```

Comment:
Please keep the --config and --section options so that the config file can still be used.